scripts/exp_citation.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
```

Comment:
it will be great if we use exp.sh. Again, another PR with the suggestions mentioned above.

Reply:
yes, I'll remove this to keep the codebase cleaner.